protocol/bin/flow.js

```diff
@@ -376,3 +381,60 @@ type CommonResponseHandler = {
   const toWrite = [typePrelude, codeGenerators.channelMapPrelude, typeDefs.join('\n\n'), incomingMap].join('\n')
   fs.writeFileSync(project.out, toWrite)
 }
+
+function decapitalize (s) {
+  return s.charAt(0).toLowerCase() + s.slice(1)
```

Comment:
Although it would change TLF into Tlf, and ID into Id, so we might need to make special case for them if that's not what we want